plugins/outputs/http/http_test.go

```diff
@@ -0,0 +1,100 @@
+package http
```

Comment:
I think the "net / http / httptest" package is suitable for testing Handler. The http output I implement is to send the metric collected from the input to the endpoint in the HTTP Spec, which is different from the Handler implementation. So, what do you think about trying to write a test like this?

In the http output, the server that returns the requested request body is displayed and sends a request to the server. Then, the test code is written with the body included in the response of the corresponding mock server.

Client => Mock Echo Server (This server sends the requested body as response.)